addons/mail/static/src/models/thread/thread.js

```diff
@@ -2232,6 +2240,12 @@ function factory(dependencies) {
         hasActivities: attr({
             default: false,
         }),
+        /**
+         * Determines whether the buttons to start a RTC call should be displayed.
+         */
+        hasCallButtons: attr({
```

Comment:
I would suggest to name the field `hasCallFeature` rather than `hasCallButtons` on thread model, as "buttons" is a very view-specific term.